### PR TITLE
Resolves #2250

### LINKED ISSUE #2250: Example 1 command should be the command for example 2 and vice-versa
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: dcb633a0-9800-2db6-5d5e-82f8852c1316
* Version Independent ID: 4700a39a-5694-1d2d-e166-354704cd19f3
* Content: [Install-ADDSDomainController (ADDSDeployment)](https://docs.microsoft.com/en-us/powershell/module/addsdeployment/install-addsdomaincontroller?view=win10-ps)
* Content Source: [docset/windows/addsdeployment/Install-ADDSDomainController.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/windows/addsdeployment/Install-ADDSDomainController.md)
* Product: **w10**
* Technology: ****
* GitHub Login: @Kateyanne
* Microsoft Alias: **v-kaunu**

### PR DESCRIPTION
**Pull request by velkovb**
Fix example 1 and 2
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2250

### CODE REVIEW

File: docset/winserver2012r2-ps/addsdeployment/Install-ADDSDomainController.md [hunk 1]
```diff
@@ -637,14 +625,14 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: `-Debug`, `-ErrorAction`, `-ErrorVariable`, `-InformationAction`, `-InformationVariable`, `-OutVariable`, `-OutBuffer`, `-PipelineVariable`, `-Verbose`, `-WarningAction`, and `-WarningVariable`. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216)..
```
Comment (velkovb) [inline docset/winserver2012r2-ps/addsdeployment/Install-ADDSDomainController.md:628]:
```suggestion
This cmdlet supports the common parameters: `-Debug`, `-ErrorAction`, `-ErrorVariable`, `-InformationAction`, `-InformationVariable`, `-OutVariable`, `-OutBuffer`, `-PipelineVariable`, `-Verbose`, `-WarningAction`, and `-WarningVariable`. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: docset/winserver2012r2-ps/addsdeployment/Install-ADDSDomainController.md [hunk 2]
```diff
@@ -556,9 +544,9 @@ Accept wildcard characters: False
...
 Specifies that only a base set of validations will be performed.
-This behavior is equivalent to the validations that were performed when using Dcpromo.exe in earlier versions of Windows Server to add a new domain controller.
+This behavior is equivalent to the validations that were performed when using `Dcpromo.exe` in earlier versions of Windows Server to add a new domain controller.
 When this switch parameter is set, it specifies that additional preliminary checks should be bypassed.
-For more information on the scope of these additional preliminary checks that the ADDSDeployment module performs by default when using Windows Server 2012, refer to the table in the section "Prerequisite Checking" in the Understand and Troubleshoot AD DS Simplified Administration in Windows Server 2012 guide (http://go.microsoft.com/fwlink/?LinkID=237244).
+For more information on the scope of these additional preliminary checks that the ADDSDeployment module performs by default when using Windows Server 2012R2, refer to the table in the section "ADPrep and Prerequisite Checking Architecture" in the [AD DS Simplified Administration](https://go.microsoft.com/fwlink/?LinkID=237244).
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/addsdeployment/Install-ADDSDomainController.md:549]:
```suggestion
For more information on the scope of these additional preliminary checks that the ADDSDeployment module performs by default when using Windows Server 2012 R2, refer to the table in the section "ADPrep and Prerequisite Checking Architecture" in the [AD DS Simplified Administration](https://go.microsoft.com/fwlink/?LinkID=237244).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - one minor edit.

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by rjagiewich** [comment]
Closing and reopening to see if this helps validation/build

**Pull request merged**